recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "openmp" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "db55d85a7bb289804dc42fc5c8e35ca24dfc3885782261b675a194fd7e206e26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://releases.llvm.org/{{ version }}/openmp-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Forgot to mention, please add 

```yaml
  skip: true  # [win and py35]
```

Since both Python 3.5 and 3.6 use the same `vc` version, we only need to build for one of them.